src/features/profil/components/ProfilPage.tsx

```
@@ -19,10 +18,7 @@ const ProfilPage = (props: PageProps) => {
         <title>{metatags.createTitle(config.title)}</title>
       </Head>
       <PageLayout>
-        <>
-          {/* <PageHeader {...config} backArrow={props.backArrow} /> */}
-          {props.children}
-        </>
+        <>{props.children}</>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)